github/repos_traffic.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Referrer represent a referrer information.
+type Referrer struct {
+	Referrer *string `json:"referrer,omitempty"`
+	Count    *int    `json:"count,omitempty"`
+	Uniques  *int    `json:"uniques,omitempty"`
+}
+
+// Path represent a referrer information.
+type Path struct {
+	Path    *string `json:"path,omitempty"`
+	Title   *string `json:"title,omitempty"`
+	Count   *int    `json:"count,omitempty"`
+	Uniques *int    `json:"uniques,omitempty"`
+}
+
+// Time represents a time stamp as in a datapoint.
+//
+// This is necessary because github uses milliseconds timestamp.
+type Time struct {
+	time.Time
+}
+
+// UnmarshalJSON parse unix timestamp.
+func (t *Time) UnmarshalJSON(b []byte) error {
+	s := string(b)
+	i, err := strconv.ParseInt(s, 10, 64)
+	if err != nil {
+		return err
+	}
+	// We can drop the reaminder as returned values are days and it will always be 0
+	*t = Time{time.Unix(i/1000, 0)}
+	return nil
+}
+
+// Datapoint represent a view in views on clones.
+type Datapoint struct {
+	Timestamp *Time `json:"timestamp,omitempty"`
+	Count     *int  `json:"count,omitempty"`
+	Uniques   *int  `json:"uniques,omitempty"`
+}
+
+// Views represent information about views on the last 14 days.
+type Views struct {
+	Views   *[]Datapoint `json:"views,omitempty"`
+	Count   *int         `json:"count,omitempty"`
+	Uniques *int         `json:"uniques,omitempty"`
+}
+
+// Clones represent information about clones on the last 14 days.
+type Clones struct {
+	Clones  *[]Datapoint `json:"clones,omitempty"`
+	Count   *int         `json:"count,omitempty"`
+	Uniques *int         `json:"uniques,omitempty"`
+}
+
+// ListReferrers list the top 10 referrers over the last 14 days.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/traffic/#list-referrers
+func (s *RepositoriesService) ListReferrers(owner, repo string) ([]*Referrer, *Response, error) {
```

Comment:
Same concern as above, but now on the methods.  I wonder if this should be `ListTrafficReferrers` (and similar naming style for methods below).  The fact that the URL for this is `/repos/o/r/traffic/popular/referrers` rather than simply `/repos/o/r/referrers` is a good indicator that maybe we should rename this.